.github/workflows/fast-dispatch-full-regressions-and-models-impl.yaml

```diff
@@ -167,7 +167,12 @@ jobs:
           source ${{ github.workspace }}/python_env/bin/activate
           cd $TT_METAL_HOME
           export PYTHONPATH=$TT_METAL_HOME
-          pytest -n auto tests/nightly/single_card/${{ matrix.model }}
+          if [[ "${{ matrix.model }}" == *"llama3"* ]]; then
+            pytest -n auto tests/nightly/single_card/llama3 -k ${{ matrix.model }}
```

Comment:
is it necessary to introduce this if statement here?

Reply:
Yes.
The end goal is to have multiple llama3 models running on fast dispatch pipeline - 1B, 3B, 8B, but we're getting out of memory on host errors when pytest runs too many tests in sequence :/

We have a pytest file that runs all llama fast dispatch tests. This test is parameterized by the name of the model: so this way if our model matrix has `llama-1b, llama-3b, etc...` we can run the test with `-k "name_of_model"` and it will run all the fast dispatch tests of said test.

This saves us from having a folder of multiple symlink tests for each llama3 model (too many redundant copies).

Reply:
let's leave a comment in the YAML file saying that this should be split later in another tt-metal issue then.

we can merge this in after and revisit this issue later

Reply:
Done. Also reverted back in the tests I commented out in T3k.